src/models/product.dao.js

```diff
@@ -0,0 +1,20 @@
+const { appDataSource } = require("./data-source");
+
+const getNewProducts = async (orderBy, limitNum) => {
+  const newProducts = await appDataSource.query(
+    `
+    SELECT 
+      products.name_en as enName,
+      products.name_ko koName,
+      product_options.price,
+      sizes.name as size,
+      products.created_at
+    FROM products INNER JOIN product_options 
+    ON products.id = product_id INNER JOIN sizes ON sizes.id = size_id
+    ORDER BY ${orderBy} limit ${limitNum};
+    `
+  );
```

Comment:
```suggestion
  const newProducts = await appDataSource.query(
    `
    SELECT 
      products.name_en as enName,
      products.name_ko koName,
      product_options.price,
      sizes.name as size,
      products.created_at
    FROM products INNER JOIN product_options 
    ON products.id = product_id INNER JOIN sizes ON sizes.id = size_id
    ORDER BY ? limit ?;
    `, [ orderBy, limitNum ]
  );
```
위와 같이 만들어진 객체의 .query method에는 SQL 문이 들어갈 수 있습니다.

그 안에 작성되는 SQL 문에는 작성해주신 것과 같이 자리 표시자(${ })를 통해 값을 넣을 수도, 혹은 제가 suggestion으로 작성드린 SQL 안에 ? 를 지정하여 동적으로 쿼리문이 수행될 수 있도록 하는 Placeholder를 사용하는 방법으로 값을 넣을 수도 있습니다.

두가지 방법 모두 다 SQL 안에 입력값을 넣어줄 수 있습니다.

다만 제가 작성한 것과 같이 ?를 활용하여 Placeholder를 사용하는 것을 지향해야 하는 이유는
보안적인 측면과 관련있습니다.

백엔드는 비단 클라이언트와 데이터베이스 사이의 접점이라고 볼 수 있는 API를 단순 동작하도록 구현하는 것 뿐만이 아닌, Data를 주고 받을 때, Database에서 원하는 data를 추출할 때를 포함한 여러 과정에서의 보안성도 고려해야 합니다.

해당 리뷰와 가장 관련이 있는 악성 공격으로는 SQL Injection 이라는 것이 있고 이는 정상적인 사용자가 아닌 누군가가 SQL문에 중요한 데이터를 추출할 수 있는 값을(ex. 항상 TRUE가 될 수 있는) 의도적으로 삽입 하여 data를 탈취하는 공격이고, ${}를 사용하면 이에 취약한 코드가 된다고 볼 수 있습니다.

따라서 가능하면 ?를 사용하고 placeholder를 통해 값을 넣어주는 방식으로 구현해주세요!